secure/key-manager/src/tests.rs

```diff
@@ -163,10 +197,13 @@ impl LibraInterface for TestLibraInterface {
     }
 }
 
+// TODO(davidiw): Notify external upon reconfigure and update epoch via ledger_info and also add
+// next validator set if appropriate...
```

Comment:
do we need this?

Reply:
Well the executor isn't really functionally complete...